SumoLogic.Logging.Common.Tests/Sender/SumoLogicMessageSenderTest.cs

```diff
@@ -69,7 +76,7 @@ public SumoLogicMessageSenderTest()
         public void HttpClientCallWith200ResponseTest()
         {
             Assert.Equal(0, this.messagesHandler.ReceivedRequests.Count);
-            this.sumoLogicMessageSender.Send("name", "body");
+            this.sumoLogicMessageSender.Send("name");

```

Comment:
Or rather, looks like the code previously had a minor bug where it was using `"name"` and the body, and `"body"` as the name